src/pivot-table/components/cells/DimensionValue/__tests__/DimensionValue.test.tsx

```diff
@@ -0,0 +1,486 @@
+import type { stardust } from "@nebula.js/stardust";
```

Comment:
Just moved. I think all tests except one is still here. But some `expect` steps have been removed since the verified the styling. That is now done via `get-style.test.ts`